Robot2019/src/main/java/frc/robot/commands/IntakeCargo.java

```diff
@@ -11,22 +11,26 @@
 import edu.wpi.first.wpilibj.command.Command;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import frc.robot.subsystems.Cargo;
-
+import frc.robot.subsystems.Lights;
+import frc.robot.commands.SetLight;
 public class IntakeCargo extends Command {
   Timer timer;
   Cargo cargo;
+  Lights lights;
   boolean overdraw;
 
-  public IntakeCargo(Cargo cargo) {
+  public IntakeCargo(Cargo cargo, Lights lights) {
     requires(cargo);
     this.cargo = cargo;
+    this.lights = lights;
     timer = new Timer();
     overdraw = false;
   }
 
   @Override
   protected void initialize() {
     timer.reset();
+    SetLight signalCargo = new SetLight(lights, Lights.LightState.CARGO);
```

Comment:
This doesn't actually run the SetLight() command, it just creates one. Regardless, why not just call lights.setLights(Lights.LightState.CARGO))?